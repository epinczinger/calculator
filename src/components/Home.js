import React from 'react';

function Home() {
  return (
    <div className="text-center px-2 text-gray-800 sm:animate-pulse">
      <h1 className="py-4 text-lg font-bold">
        Welcome to
        {' '}
        <span className="text-yellow-400">Math-magicians</span>
      </h1>
      <p className="">
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
        veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
        commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
        velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
        occaecat cupidatat non proident, sunt in culpa qui officia deserunt
        mollit anim id est laborum.
      </p>
    </div>
  );
}

export default Home;
