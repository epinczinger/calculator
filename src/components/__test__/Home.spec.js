import React from "react";
import Home from "../Home";
import renderer from "react-test-renderer";

describe("The Home", () => {
  it("renders as expected matching snapshot", () => {
    const tree = renderer
      .create(<Home />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
