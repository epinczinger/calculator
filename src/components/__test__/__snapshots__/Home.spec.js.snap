// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Home renders as expected matching snapshot 1`] = `
<div
  className="text-center px-2 text-gray-800 sm:animate-pulse max-w-md mx-auto"
>
  <h1
    className="py-4 text-lg lg:text-2xl font-bold"
  >
    Welcome to
     
    <span
      className="text-yellow-400"
    >
      Math-magicians
    </span>
  </h1>
  <p
    className=""
  >
    A mathemagician is a mathematician who is also a magician. The term
    <q>
      mathemagic
    </q>
     
    is believed to have been introduced by Royal Vale Heath with his 1933 book Mathemagic.. The name
    <q>
      mathemagician
    </q>
     
    was probably first applied to Martin Gardner, but has since been used to describe many mathematician/magicians, including Arthur T. Benjamin, Persi Diaconis, and Colm Mulcahy. Diaconis has suggested that the reason so many mathematicians are magicians is that
     
    <q>
      inventing a magic trick and inventing a theorem are very similar activities.
    </q>
     
    Mathemagician is a neologism, specifically a portmanteau, that combines mathematician and magician. A great number of self-working mentalism tricks rely on mathematical principles. Max Maven often utilizes this type of magic in his performance. The Mathemagician is name of a character in the 1961 children's book The Phantom Tollbooth. He is the ruler Digitopolis, the kingdom of mathematics.
  </p>
</div>
`;
